src/main/tut/2.4-representable-functors.md

```diff
@@ -0,0 +1,208 @@
+```Haskell
+C(a, x) -> C(a, y)
+```
+```scala
+`C(A, X)` => `C(A, Y)`
+```
+................
+```Haskell
+f ∘ h :: a -> y
+```
+```scala
+val `f ∘ h`: A => Y
+```
+................
+```Haskell
+type Reader a x = a -> x
+```
+```tut:silent
+case class Reader[A, X](run: A => X)
+```
```

Comment:
Strictly speaking the haskell code is using a type synonym, not a newtype, so the translation is a also a type synonym, not a case class. Not sure this is worth changing though

Reply:
I've shamefully copy-pasted it from the 1.10 and not noticed the difference.
It was `newtype Reader e a = Reader (e -> a)`

Reply:
right, I don't really mind too much which way you go, just wanted to point out the discrepancy :)